src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1495,11 +1496,11 @@ internal override void Check(IHostEnvironment env)
             _outputColumns = outCols.ToArray();
         }
 
-        internal SdcaBinaryTrainer(IHostEnvironment env, Arguments args,
+        internal SdcaBinaryTrainer(IHostEnvironment env, Options options,
             string featureColumn, string labelColumn, string weightColumn = null)
```

Comment:
> string featureColumn, string labelColumn, string weightColumn = null [](start = 11, length = 69)

are those still needed? can they be passed to options?

Reply:
This was kinda weird. If you see there is another constructor  below in Line 1540 .

For now I am making all constructors internal.  All constructor cleanup will be part of issue  #2100

---
In reply to: [248021483](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248021483) [](ancestors = 248021483)